### PR TITLE
fix: some changes

### PR DESCRIPTION
**Pull request by Srish-ty**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by Srish-ty** [comment]
@Shurtu-gal , please review

**Comment by Srish-ty** [comment]
@Shurtu-gal please review

File: api/src/apollo/Event/mutations.ts [hunk 1]
```diff
@@ -10,28 +10,27 @@ export const EventCreateInputType = inputObjectType({
...
+    t.int('minTeamSize');
   },
 });
 
 export const createEvent = mutationField('createEvent', {
   type: 'Event',
   description: `Creates a new event record`,
-  authorize: (_parent, args, ctx) =>
+  authorize: _parent => true,
+  /* authorize: (_parent, args, ctx) =>
```
Comment (dsc-nitr) [inline api/src/apollo/Event/mutations.ts:33]: Add permissions here as well.

File: api/src/helpers/auth/checkPermissions.ts [hunk 2]
```diff
@@ -96,11 +98,15 @@ const checkPermissions = async ({
     const permissions = await PermissionModel.findOne({
       uid: decodedToken.uid,
     });
+    console.log(permissions);
```
Comment (dsc-nitr) [inline api/src/helpers/auth/checkPermissions.ts:101]: nit: remove logs

**Review by dsc-nitr — CHANGES_REQUESTED** [review]

File: api/src/apollo/Event/mutations.ts (hunk 1 above)
Comment (Srish-ty) [reply api/src/apollo/Event/mutations.ts]: done. https://github.com/dscnitrourkela/project-dates/pull/160/commits/10aeecccc753efa066166e6a29cefb82de7d0ae2

File: api/src/helpers/auth/checkPermissions.ts (hunk 2 above)
Comment (Srish-ty) [reply api/src/helpers/auth/checkPermissions.ts]: removed. https://github.com/dscnitrourkela/project-dates/pull/160/commits/10aeecccc753efa066166e6a29cefb82de7d0ae2

**Review by dsc-nitr — APPROVED** [review]

**Pull request merged**